applications/ansys_mechanical_desktop/metadata.md

```diff
@@ -3,5 +3,9 @@
 id: "ansys_mechanical_vdi_application" # needs to be **unique** per application, changing results in a new application
 name: "Ansys Mechanical (VDI)"
 category: "SIMULATION"
+tags:
+- Ansys
+- FEA
+- Desktop
```

Comment:
```suggestion
- desktop
```
4/7 desktop tags are lower case and it seems you are going for all lower case on anything that is not an acronym or proper noun?